ArgumentResolver/ServiceRequestResolver.php

```diff
@@ -55,15 +55,15 @@ public function __construct(
     /**
       * {@inheritdoc}
      */
-    public function supports(Request $request, ArgumentMetadata $argument)
+    public function supports(Request $request, ArgumentMetadata $argument): bool
```

Comment:
The method signature does not match the Interface signature for Symfony < 6.0

Reply:
yes, thanks, will check in other bundles as well...

Reply:
It will work if interface doesn't have return type definition
![image](https://github.com/auto1-oss/service-api-handler-bundle/assets/39693338/556d8559-b710-4d27-aae8-726923a1efa5)